zebra-test/src/net.rs

```diff
@@ -1,7 +1,49 @@
 //! Network testing utility functions for Zebra.
 
+use std::env;
+
 use rand::Rng;
 
+/// The name of the env var that skips Zebra tests which need reliable,
+/// fast network connectivity.
+///
+/// We use a constant so that the compiler detects typos.
+const ZEBRA_SKIP_NETWORK_TESTS: &str = "ZEBRA_SKIP_NETWORK_TESTS";
+
+/// The name of the env var that skips Zebra's IPv6 tests.
+///
+/// We use a constant so that the compiler detects typos.
+const ZEBRA_SKIP_IPV6_TESTS: &str = "ZEBRA_SKIP_IPV6_TESTS";
+
+/// Should we skip Zebra tests which need reliable, fast network connectivity?
+//
+// TODO: separate "good and reliable" from "any network"?
+pub fn zebra_skip_network_tests() -> bool {
+    if env::var_os(ZEBRA_SKIP_NETWORK_TESTS).is_some() {
+        // This message is captured by the test runner, use
+        // `cargo test -- --nocapture` to see it.
+        eprintln!("Skipping network test because '$ZEBRA_SKIP_NETWORK_TESTS' is set.");
+        return true;
+    }
+
+    false
+}
+
+/// Should we skip Zebra tests which need a local IPv6 network stack and
+/// IPv6 interface addresses?
+///
+/// If we are skipping network tests, we also skip IPv6 tests.
```

Comment:
```suggestion
/// Since `zebra_skip_network_tests` only disables tests which need reliable network connectivity,
/// we allow IPv6 tests even when `ZEBRA_SKIP_NETWORK_TESTS` is set.
```